multiversx_sdk/abi/abi_definition.py

```diff
@@ -87,7 +87,7 @@ def __init__(self,
                  outputs: List["ParameterDefinition"],
                  payable_in_tokens: List[str],
                  only_owner: bool,
-                 title: str) -> None:
+                 title: Optional[str]) -> None:
```

Comment:
If optional, can be set to None by default. If None by default, then, in constructor, do `self.title = title or ""`.

Otherwise, here it can also be `title: str = ""` (and no other change in constructor).

Reply:
Done